packages/terra-application-utility/src/_Utils.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+
+const KEY_CODES = {
+  ENTER: 13,
+  SPACE: 32,
+  UP_ARROW: 38,
+  DOWN_ARROW: 40,
+  LEFT_ARROW: 37,
+  RIGHT_ARROW: 39,
+  BACK_SPACE: 8,
+};
+
+const LOCATIONS = {
```

Comment:
are locations still supported?

Reply:
Yes, this is used to specify if a menu item is a list item or footer button.

Reply:
We should add usage doc to `terra-framework/packages/terra-application-utility/docs/` on this util. There we can document how this is used to specify if a menu item is a list item or footer button and shape of `itemShape`. Otherwise, I think users will likely be confused about using this.